requirments.txt

```diff
@@ -0,0 +1,2 @@
+#install the Font Awesome package
+npm install @fortawesome/react-fontawesome @fortawesome/free-solid-svg-icons
```

Comment:
I think we don't need this file. Since you listed these packages as dependencies in package.json, they should be installed with running just `npm install` as it's instructed in the readme file. Or am I missing something?

Reply:
You are right, but I added this file to mention about the package that installed, incase someone missed to run npm install.

should i delete it ?

Reply:
They would not need it, as a simple `npm install` would install this since the package got added to the package.json file

Reply:
ok i will remove it :)